package/MDAnalysis/transformations/wrap.py

```diff
@@ -36,8 +36,10 @@
 
 from ..lib._cutil import make_whole
 
+from .base import TransformationBase
```

Comment:
Does the new dimensions transformation need to be updated too?

Reply:
Updated:)